pip/exceptions.py

```diff
@@ -39,13 +43,209 @@ class PreviousBuildDirError(PipError):
     """Raised when there's a previous conflicting build directory"""
 
 
-class HashMismatch(InstallationError):
-    """Distribution file hash values don't match."""
-
-
 class InvalidWheelFilename(InstallationError):
     """Invalid wheel filename."""
 
 
 class UnsupportedWheel(InstallationError):
     """Unsupported wheel."""
+
+
+# The recommended hash algo of the moment. Feel free to change this any time.
```

Comment:
Changing it won't break compatibility with anything. (I'll clarify the comment.)